core/capabilities/syncer.go

```diff
@@ -0,0 +1,79 @@
+package capabilities
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+
+	"github.com/smartcontractkit/libocr/ragep2p"
+	ragetypes "github.com/smartcontractkit/libocr/ragep2p/types"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	p2ptypes "github.com/smartcontractkit/chainlink/v2/core/services/p2p/types"
+)
+
+type registrySyncer struct {
```

Comment:
can you add a comment about what the registry syncer does? it work be ideal the go doc explains every capability and what function it serves